cypress/features/regression/experimental/dateRange.feature

```diff
@@ -1,42 +1,30 @@
 Feature: Date Range component
-  I want to change Date Range component properties
-
-  Background: Open Date Range component page
-    Given I open "Experimental Date Range" component page
+  I want to check Date Range component properties
 
   @positive
   Scenario Outline: Change Date Range start label to <label>
-    When I set startLabel to <label> word
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "<nameOfObject>" object name
     Then startLabel on preview is <label>
     Examples:
-      | label                   |
-      | mp150ú¿¡üßä             |
-      | !@#$%^*()_+-=~[];:.,?{} |
-  # @ignore because of FE-2782
-  # | &"'<>|
+      | label                   | nameOfObject               |
+      | mp150ú¿¡üßä             | startLabelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | startLabelSpecialCharacter |
 
   @positive
   Scenario Outline: Change Date Range end label to <label>
-    When I set endLabel to <label> word
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "<nameOfObject>" object name
     Then endLabel on preview is <label>
     Examples:
-      | label                   |
-      | mp150ú¿¡üßä             |
-      | !@#$%^*()_+-=~[];:.,?{} |
-  # @ignore because of FE-2782
-  # | &"'<>|
+      | label                   | nameOfObject             |
+      | mp150ú¿¡üßä             | endLabelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | endLabelSpecialCharacter |
 
   @positive
   Scenario: Enable labels inline checkbox
-    When I set startLabel to "label"
-      And I set endLabel to "label"
-      And I check labelsInline checkbox
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "labelsInline" object name
```

Comment:
you have here `labelSinline` and below `labelInlineFalse` in the first variant:
label**S** and in the second label. It should be the same in one component

Reply:
fixed